bzt/modules/python.py

```diff
@@ -37,18 +38,18 @@
 IGNORED_LINE = re.compile(r"[^,]+,Total:\d+ Passed:\d+ Failed:\d+")
 
 
-class NoseTester(SubprocessedExecutor, HavingInstallableTools):
+class ApiritifNoseExecutor(SubprocessedExecutor, WidgetProvider):
```

Comment:
WidgetProvider is not needed here, since SubprocessedExecutor already implements it